README.md

```diff
@@ -202,15 +202,25 @@ poetry run flask drop-db
 
 ## Migrations
 
+If you have added new mapped dataclasses or modified existing ones, a migration script needs to be added.
```

Comment:
Maybe add that this should be done once the changes are somewhat final (e.g. at the end of a PR) to minimize the number of migrations.